Masterplan/Controls/DefaultTextBox.cs

```diff
@@ -98,7 +98,7 @@ protected override void OnLeave(EventArgs e)
         /// <param name="e">Event arguments.</param>
         protected override void OnKeyDown(KeyEventArgs e)
         {
-            if ((e.Modifiers & Keys.Control) == Keys.Control)
+            if ((e.Modifiers & Keys.Control) == Keys.Control & (e.Modifiers & Keys.Alt) != Keys.Alt)
```

Comment:
`&` needs to be `&&` (logical and)

Also I'd prefer the two conditions to be in parentheses. So: `(XXX) && (XXX)`